client/src/plus/deep-dives/overview/index.scss

```diff
@@ -6,196 +6,167 @@
-        margin: layout.$base-spacing 0;
-      }
+    @media #{layout.$mq-small-desktop-and-up} {
+      margin-top: -24px;
     }
 
-    h2 {
-      background: theme.$gradient-accent;
-      background-clip: text;
-      color: theme.$mdn-green400;
-      font-size: 18px;
-      line-height: 160%;
-      margin-top: 0;
-      text-transform: uppercase;
+    @media #{layout.$mq-large-desktop-and-up} {
+      margin-top: -56px;
     }
 
-    a {
-      &:link,
-      &:visited {
-        color: theme.$white;
-        text-decoration: none;
-      }
-
-      &:hover,
-      &:focus {
-        text-decoration: underline;
-      }
-    }
-
-    p {
-      font-size: typography.$default-font-size;
-      margin-bottom: 0;
-      max-width: 450px;
-
-      @media #{layout.$mq-small-desktop-and-up} {
-        font-size: typography.$small-medium-font-size;
-      }
+    h1 {
+      font-size: 36px;
+      line-height: typography.$heading-line-height;
     }
   }
 
-  p.author,
-  .author-links {
-    font-size: 24px;
-  }
-
   .author {
     font-weight: bold;
     line-height: 150%;
   }
-
-  .author-links {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-
-    li {
-      display: inline-block;
-      margin-right: math.div(layout.$base-spacing, 4);
-    }
-  }
 }
 
 .overview-content-container {
   padding: 0 layout.$base-spacing math.div(layout.$base-spacing, 2);
 
-  @media #{layout.$mq-small-desktop-and-up} {
-    padding: 0 120px 36px;
-  }
-
-  @media #{layout.$mq-large-desktop-and-up} {
-    margin: 36px auto layout.$base-spacing;
-    max-width: layout.$max-width-default;
-  }
-
-  h3,
-  h4 {
-    font-size: 24px;
-    margin: 36px 0;
+  h2 {
+    font-size: 14px;
+    margin-bottom: 16px;
+    text-transform: uppercase;
   }
 
-  h4 {
+  h3 {
     font-size: 18px;
-    margin: 36px 0;
+    margin: 16px 0;
+
+    @media #{layout.$mq-small-desktop-and-up} {
+      margin-top: 0;
+    }
   }
 
   p {
     font-size: 18px;
     line-height: 200%;
   }
 
-  li {
-    margin-bottom: layout.$base-spacing;
+  button.ghost {
+    color: theme.$mdn-neutral900;
+    font-size: 16px;
+    font-weight: bold;
+    margin: 8px 0;
+    padding: 0;
+    text-decoration: none;
   }
 
-  a.overview-series-list-item {
-    border-radius: 4px;
-    display: block;
-    min-height: 140px;
-    padding: math.div(layout.$base-spacing, 4);
-    position: relative;
+  .show-contents-button {
+    &::before {
+      background: transparent url("./assets/plus.svg") center center no-repeat;
+      background-size: 18px;
+      content: "";
+      display: inline-block;
+      height: 18px;
+      margin-right: 8px;
+      vertical-align: sub;
+      width: 18px;
+    }
+  }
 
+  a.overview-series-list-item {
     &:link,
     &:visited {
       color: theme.$mdn-neutral900;
     }
+  }
 
-    &:hover,
-    &:focus {
-      background-color: theme.$mdn-neutral100;
-      text-decoration: none;
+  .series-item {
```

Comment:
I wanted the table of contents to be under the image like this 
![image](https://user-images.githubusercontent.com/44966503/129598504-17645bf9-66aa-4412-af0c-80cac99d80d9.png)
instead of on the side 
![image](https://user-images.githubusercontent.com/44966503/129598463-cecbc6d3-a050-48db-8c85-38c9e9871046.png) but I can't figure out a fast way to change that